erdpy/cli_validators.py

```diff
@@ -51,9 +51,10 @@ def setup_parser(subparsers: Any) -> Any:
     _add_nodes_arg(sub)
     sub.set_defaults(func=do_unstake_nodes)
 
-    sub = cli_shared.add_command_subparser(subparsers, "validator", "unstake-tokens", "It will unStake tokens, if "
-                                                                                      "provided value is bigger that "
-                                                                                      "topUp value will unStake nodes")
+    sub = cli_shared.add_command_subparser(subparsers, "validator", "unstake-tokens", "UnStakeToken will unStakes the "
```

Comment:
This command will un-stake the given amount (if value is greater than the existing topUp value, it will unStake one or several nodes)

Reply:
done.